e2etest/container/container.go

```diff
@@ -160,6 +171,16 @@ func (m *Manager) RunBitcoindResource(
 				"-fallbackfee=0.0002",
 			},
 		},
+		func(config *docker.HostConfig) {
+			config.PortBindings = map[docker.Port][]docker.PortBinding{
+				"8332/tcp":  {{HostIP: "", HostPort: strconv.Itoa(randomAvailablePort(t))}},
+				"8333/tcp":  {{HostIP: "", HostPort: strconv.Itoa(randomAvailablePort(t))}},
+				"28332/tcp": {{HostIP: "", HostPort: strconv.Itoa(randomAvailablePort(t))}},
+				"28333/tcp": {{HostIP: "", HostPort: strconv.Itoa(randomAvailablePort(t))}},
+				"18443/tcp": {{HostIP: "", HostPort: strconv.Itoa(randomAvailablePort(t))}},
+				"18444/tcp": {{HostIP: "", HostPort: strconv.Itoa(randomAvailablePort(t))}},
```

Comment:
as we will be running multiple instances of both bitcoind and Babylon, ensure we use random ports so they don't clash

Reply:
just curious, why would we be running multiple instances? For e2e in vigilante, a single pair of bitcoind and babylond should be enough, no?

Reply:
Possibly, but I feel like e2e tests should be isolated to confirm state, rather than tests possibly conflicting with one another

Reply:
This means that when we start running tests with `t.Parallel()` we will be spawning bitcoind and Babylon container per e2e tests, this ensures that ports won't conflict. There's no need for these tests to be sequential if we can run them in parallel